src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -1124,13 +1125,6 @@ public Void visitSwitchExpr(JsoniqParser.SwitchExprContext ctx) {
 
     @Override
     public Void visitSwitchCaseClause(JsoniqParser.SwitchCaseClauseContext ctx) {
```

Comment:
This method too is now empty and can be removed.

Reply:
DONE.